Modules/DevTools/TestFramework/TestRunner/src/ALTestTool.Page.al

```diff
@@ -304,11 +305,72 @@ page 130451 "AL Test Tool"
                         TestMethodLine: Record "Test Method Line";
                         TestSuiteMgt: Codeunit "Test Suite Mgt.";
                     begin
+                        if GuiAllowed() then begin
+                            if not Confirm(DeleteQuestion, false) then
+                                exit;
+                        end;
+
                         CurrPage.SetSelectionFilter(TestMethodLine);
                         TestMethodLine.DeleteAll(true);
                         TestSuiteMgt.CalcTestResults(Rec, Success, Failure, Skipped, NotExecuted);
                     end;
                 }
+                action(InvertRunCheckLines)
+                {
+                    ApplicationArea = All;
+                    Caption = '&Invert Run Check';
+                    Image = Change;
+                    Promoted = true;
+                    PromotedCategory = Process;
+                    PromotedIsBig = true;
+                    PromotedOnly = true;
+                    ToolTip = 'Invert Run Check on selected lines.';
+
+                    trigger OnAction()
```

Comment:
I think all of this can be done with and handful of lines of code ;-)
        
CurrPage.SetSelectionFilter(TestMethodLine);

if TestMethodLine.FindSet(true, false) then
    repeat
        TestMethodLine.Validate(Run, not TestMethodLine.Run);
        TestMethodLine.Modify(true);
    until TestMethodLine.Next() = 0;

There is no need to ask questions or show status - this is a low risk, fast operation. You don't need to fix this. I'll do that with my final check-in. Just wanted to update you :-)